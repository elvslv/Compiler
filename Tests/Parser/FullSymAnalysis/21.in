var i, j, n, l, k: integer;
begin
	for i := 1 to n do
	begin
		l := l + k;
		k := k + 5;
		j := 7;
	end
end.