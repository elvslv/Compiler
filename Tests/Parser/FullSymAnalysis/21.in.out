var I : INTEGER;
var J : INTEGER;
var K : INTEGER;
var L : INTEGER;
var N : INTEGER;

begin                
|                    
|___for              
|   |                
|   |___:=           
|   |   |            
|   |   |___I        
|   |   |            
|   |   |___1        
|   |                
|   to               
|   |                
|   |___N            
|   |                
|   do               
|   |                
|   |___begin        
|       |            
|       |___:=       
|       |   |        
|       |   |___L    
|       |   |        
|       |   |___+    
|       |       |    
|       |       |___L
|       |       |    
|       |       |___K
|       |            
|       |___:=       
|       |   |        
|       |   |___K    
|       |   |        
|       |   |___+    
|       |       |    
|       |       |___K
|       |       |    
|       |       |___5
|       |            
|       |___:=       
|       |   |        
|       |   |___J    
|       |   |        
|       |   |___7    
|       |            
|       end          
|                    
end                  
